.github/workflows/release.yml

```
@@ -204,9 +204,11 @@ jobs:
           make -j1 binary-linux binary-darwin binary-windows
           mkdir -p ./.releases
           cp ./output/binary/linux/bin/step ./.releases/step-linux-${{ needs.create_release.outputs.version }}
-          cp ./output/binary/linux/bin/step ./.releases/step-linux-latest-integration
+          cp ./output/binary/linux/bin/step ./.releases/step-linux-latest
           cp ./output/binary/darwin/bin/step ./.releases/step-darwin-${{ needs.create_release.outputs.version }}
+          cp ./output/binary/darwin/bin/step ./.releases/step-darwin-latest
           cp ./output/binary/windows/bin/step ./.releases/step-windows-${{ needs.create_release.outputs.version }}.exe
+          cp ./output/binary/darwin/bin/step ./.releases/step-windows-latest.exe
```

This typo, otherwise it looks goot to me
```suggestion
          cp ./output/binary/windows/bin/step ./.releases/step-windows-latest.exe
```